@uportal/user-profile-menu/.gitignore

```diff
@@ -0,0 +1,21 @@
+.DS_Store
```

Comment:
This gitignore probably isn't needed.
The top level gitignore covers the same files.